cli/bench/config.go

```diff
@@ -0,0 +1,48 @@
+package bench
+
+import (
+	"fmt"
+
+	"github.com/FZambia/tarantool"
+	"github.com/tarantool/cartridge-cli/cli/context"
+)
+
+var bench_space_name = "__benchmark_space__"
+var bench_space_primary_index_name = "__bench_primary_key__"
+
+func print_operation_count(operations_map []Operaiton) {
+	for _, operation := range operations_map {
+		fmt.Println("	"+operation.name+" count: ", operation.count)
+	}
+}
+
+func print_operation_percent(operations_map []Operaiton) {
+	for _, operation := range operations_map {
+		fmt.Printf("\t%s: %2.0f%%\n", operation.name, operation.percent*100)
+	}
+}
+
+func print_operation_connections_count(operations_map []Operaiton) {
+	for _, operation := range operations_map {
+		fmt.Println("	"+operation.name+" connections count: ", operation.connections_count)
+	}
+}
```

Comment:
Let's use ``text/template``, and you should be able to get rid of these functions.

Reply:
see above